plugins/modules/gitlab_project.py

```diff
@@ -535,6 +560,9 @@ def main():
         infrastructure_access_level=dict(type='str', choices=['private', 'disabled', 'enabled']),
         monitor_access_level=dict(type='str', choices=['private', 'disabled', 'enabled']),
         security_and_compliance_access_level=dict(type='str', choices=['private', 'disabled', 'enabled']),
+        pages_access_level=dict(type='str', choices=['private', 'disabled', 'enabled']),
+        service_desk_enabled=dict(type='bool', default=True),
```

Comment:
I think we shouldn't define a default value here, since it would make an unexpected change on existing projects which do not have service desk feature enabled.

Reply:
I've removed it, although I think it's activated as default on a new project (cf. https://gitlab.com/gitlab-org/gitlab/-/issues/412669) There might be of course instance parameters that say otherwise, but I'm not aware of them.

But now that you have mentionned it, yes, it's best not to specify it.